src/nat/agent/prompt_optimizer/register.py

```
@@ -51,7 +51,7 @@ async def prompt_optimizer_function(config: PromptOptimizerConfig, builder: Buil
         from .prompt import mutator_prompt
     except ImportError as exc:
         raise ImportError("langchain-core is not installed. Please install it to use MultiLLMPlanner.\n"
-                          "This error can be resolve by installing nvidia-nat[langchain]") from exc
+                          "This error can be resolve by installing \"nvidia-nat[langchain]\".") from exc
```

Ruff (0.13.2)

Avoid specifying long messages outside the exception class

(TRY003)